Tools/scripts/build_ci.sh

```diff
@@ -107,12 +107,12 @@ for t in $CI_BUILD_TARGET; do
                 --enable-header-checks \
                 --check-c-compiler="$c_compiler" \
                 --check-cxx-compiler="$cxx_compiler"
-        $waf clean
-        $waf all
+        time $waf clean
+        time $waf all
```

Comment:
No, please don't do this. If you want, copy what make build already does. But I'm going to continue strongly objecting removing that line.